Server/mods/deathmatch/logic/CColPolygon.h

```diff
@@ -38,6 +38,9 @@ class CColPolygon : public CColShape
     std::vector<CVector2D>::const_iterator IterBegin() { return m_Points.begin(); };
     std::vector<CVector2D>::const_iterator IterEnd() { return m_Points.end(); };
 
+    bool SetHeight(float fFloor, float fCeil);
+    void GetHeight(float &fFloor, float &fCeil) { fFloor = m_fFloor; fCeil = m_fCeil; };
```

Comment:
- Nit: ampersand should be together with `float`
- Declare as `const`

```suggestion
    void GetHeight(float& fFloor, float& fCeil) const { fFloor = m_fFloor; fCeil = m_fCeil; };
```